src/utils/server.util.ts

```
@@ -0,0 +1,79 @@
+}
+
+export async function setupSubdomain(subdomain: string, port: number, dnsRecordId: string): Promise<string> {
+  await addRecord(subdomain, dnsRecordId);
+  await createNginxConfig(subdomain, port);
+  await createNginxSymlink(subdomain);
+  await getSSL(subdomain);
+  await restartNginx();
+  return 'Subdomain setup completed';
+}
+
+async function createNginxConfig(subdomain: string, port: number): Promise<void> {
```

ESLint

[error] Replace `subdomain:·string,·port:·number` with `⏎··subdomain:·string,⏎··port:·number⏎`

(prettier/prettier)